spec/models/search/multilingual_search_spec.rb

```diff
@@ -6,10 +6,10 @@
   let(:mission) { get_mission }
   let(:qualifier_sets) do
     {
-      response: Response.search_qualifiers(mission: mission),
-      user: User.search_qualifiers,
-      question: Question.search_qualifiers,
-      sms: Sms::Message.search_qualifiers
+      response: ResponsesSearcher.new(nil, nil, mission: mission).search_qualifiers,
+      user: UsersSearcher.new(nil, nil, nil).search_qualifiers,
```

Comment:
`UsersSearcher.new(relation: User.all, query: "x").search_qualifiers`

[ ] Change method to use named params
[ ] Default scope to nil, others no default